### PR TITLE
DOCSP-38906 Create and Update BIC ODBC Driver workaround

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
- https://jira.mongodb.org/browse/DOCSP-38906
- Stage: https://preview-mongodbkanchanamongodb.gatsbyjs.io/bi-connector/DOCSP-38906/reference/odbc-driver/
- Build Log: https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66d0965bc87522bd3a889524

### CODE REVIEW

File: source/reference/odbc-driver.txt [hunk 1]
```diff
@@ -62,8 +62,45 @@ The {+odbc-driver+} is available for:
   .. include:: /includes/temp-warn-odbc-catalina.rst
 
 - Ubuntu 14.04 and 16.04
+
 - RHEL 7
 
+Workaround for Unsupported Platforms
```
Comment (bucaojit) [inline source/reference/odbc-driver.txt:68]: Let's keep this more general, that way we will not need to update the values or the instructions if things change. 

I'd suggest something like:

- Install the MySQL Cleartext Authentication plugin for your platform.
- Download and install the Oracle MySQL ODBC driver compatible with your platform from the [MySQL Connector/ODBC Downloads page](https://dev.mysql.com/downloads/connector/odbc/).
- Prior to connecting, ensure that the Cleartext plugin is enabled.

**Review by bucaojit — CHANGES_REQUESTED** [review]
@kanchana-mongodb A more generic section would be better.  The instructions I had in BI-2883 were more for our benefit of documenting how to reproduce it.  I added a suggestion that you can update for the document.

**Review by bucaojit — APPROVED** [review]
LGTM

**Review by corryroot — APPROVED** [review]
LGTM!

**Pull request merged**